daemon/container.go

```diff
@@ -576,6 +581,64 @@ func validateID(id string) error {
 	return nil
 }
 
+func (container *Container) Checkpoint(opts *libcontainer.CriuOpts) error {
+	if err := container.daemon.Checkpoint(container, opts); err != nil {
+		return err
+	}
+
+	if opts.LeaveRunning == false {
```

Comment:
Can just be if `!opts.LeaveRunning {`
Also, shouldn't this opt be called `Shutdown` or something?